jupyter/map_analysis/comparing_osm_physsim.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pyrosm
+
+import numpy as np
+import pandas as pd
+import geopandas as gpd
+import matplotlib.pyplot as plt
+
+from pyproj import CRS, Transformer
+from geopandas import GeoDataFrame
+from shapely.geometry import Point
+from shapely.geometry.multilinestring import MultiLineString
+
+pd.set_option('display.max_rows', 500)
+pd.set_option('display.max_columns', 500)
+pd.set_option('display.width', 1000)
+
+
+# In[3]:
+
+
+# reading osm file
+
+osm_map = pyrosm.OSM("../beam_root/test/input/sf-light/r5/sflight_muni.osm.pbf")
```

Comment:
What if we create an utility method that would check for existence beam_root, and if not found then it would use the project root? That allows not to run jupyter notebooks in a container but directly on the local machine.